executor_manager/routers/routers.py

```
@@ -831,6 +833,21 @@ async def _update_validation_status_from_callback(
 
 
+async def _run_validation_task_in_background(
+    validation_task: Dict[str, Any], validation_task_id: int, image: str
+) -> None:
+    """Run validation task submission in background to avoid blocking HTTP handlers."""
+    try:
+        await asyncio.to_thread(task_processor.process_tasks, [validation_task])
+        logger.info(
+            f"Validation task submitted: task_id={validation_task_id}, image={image}"
+        )
+    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)